src/test/java/org/broadinstitute/consent/http/resources/DataRequestVoteResourceTest.java

```diff
@@ -43,5 +77,428 @@ private void initResource() {
         );
     }
 
-    // This test will be written in a future PR.
+    private User createMockUser(UserRole role, Integer dacUserId) {
+        User user = new User();
+        user.setRoles(List.of(role));
+        user.setDacUserId(dacUserId);
+        return user;
+    }
+
+    private Vote createMockVote(String voteType, Integer dacUserId, boolean voteValue) {
+        Vote vote = new Vote();
+        vote.setType(voteType);
+        vote.setDacUserId(dacUserId);
+        vote.setVote(voteValue);
+        return vote;
+    }
+
+    private DataAccessRequest createMockDAR() {
+        DataAccessRequest dar = new DataAccessRequest();
+        DataAccessRequestData darData = new DataAccessRequestData();
+        darData.setDatasetIds(List.of(1));
+        darData.setDarCode("");
+        dar.setData(darData);
+        dar.setReferenceId("");
+        return dar;
+    }
+
+    private void testCreateDataOwnerElection(Vote vote, User user, boolean returnEmptyList) throws Exception {
```

Comment:
nit: naming. Real tests should be prefixed with `test...` but helper methods should not. This one looks like it is doing a lot of setup for data owner elections, so perhaps something like `initDataOwnerElection` or something similar.